companion/index.js

```diff
@@ -111,46 +111,48 @@ processAllFiles();
 
 function sendDataToInflux(data) {
 
-    let url = `https://ay1bwnlt74.execute-api.us-east-1.amazonaws.com/test`;
-
-    //get experiment id and set empty value to "default"
-    // try {
-    //   experiment_id = JSON.parse(settingsStorage.getItem('experiment_id')).name;
-    // } catch {
-    //   console.log("experiment id not defined, setting default")
-    //   experiment_id = "default"
-    // }
+    let url = `https://0cs0bcauyc.execute-api.us-east-1.amazonaws.com/default/write-coziePublic-fitbitAPI`;
+
     let user_id = "";
     let experiment_id = "";
+    let api_key;
     // if user_id is not defined then assign the value undefined
     try {user_id = JSON.parse(settingsStorage.getItem('user_id')).name;}
     catch (error) {
         if (error instanceof TypeError) {
             user_id = "undefined"
-    }}
+        }}
 
     try {experiment_id = JSON.parse(settingsStorage.getItem('experiment_id')).name;}
     catch (error) {
         if (error instanceof TypeError) {
             experiment_id = "undefined"
-    }}
+        }}
+
+    try {api_key = JSON.parse(settingsStorage.getItem('api_key')).name}
```

Comment:
I'm guessing that this functionality is to allow people to write their own API key in the future?

Reply:
Sorry I completely forgot, we have too many versions of the same code 🙂. That is because in the new version of the code I have added a field in the settings menu where the use can enter their own API key, without the need of changing the source code.

Reply:
All good. Nice placeholder anyway. Welcome to the joys of code integration over multiple branches haha.